tabbededitor.pro

```diff
@@ -16,13 +16,17 @@ isEmpty(QTCREATOR_SOURCES):error("Error. QTC_SOURCE variable must point to Qt Cr
 IDE_BUILD_TREE = $$(QTC_BUILD)
 isEmpty(IDE_BUILD_TREE):error("Error. QTC_BUILD variable must be set.")
 
+## override IDE_LIBRARY_BASENAME for x64
+IDE_LIBRARY_BASENAME = $$(QTC_LIBRARY_BASENAME)
+isEmpty(IDE_BUILD_TREE):error("Error. QTC_IBRARY_BASENAME variable must be set to lib or lib64.")
```

Comment:
There probably should be `isempty(IDE_LIBRARY_BASENAME)` instead of `isempty(IDE_BUILD_TREE)`.